.tests/adguardhome-logs/config.yaml

```diff
@@ -0,0 +1,14 @@
+parsers:
+- crowdsecurity/syslog-logs
+- crowdsecurity/docker-logs
```

Comment:
We dont really need this for testing. If you do decide to remove the tests need to be re run